_posts/Build_TF_1.14_GPU_Ubuntu_16.0.md

```diff
@@ -0,0 +1,392 @@
+---
+layout: post
+author: "Shivam Chaudhary"
+author_url: https://github.com/andy6975
+title: "Building Tensorflow 1.14 GPU"
+subtitle: "How to build TF GPU in Ubuntu with TensorRT"
+bg_url: "https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.tensorflow.org%2F&psig=AOvVaw1Gpx9AJEqAd0jPYRa57bWQ&ust=1588994195374000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCKjGke6mo-kCFQAAAAAdAAAAABAO"
```

Comment:
This isn't opening, you can change it to [this](https://images.squarespace-cdn.com/content/v1/58cf449fe6f2e19aff94c85a/1526372369906-8QIF0U7SP7805EZWISNJ/ke17ZwdGBToddI8pDm48kNkMlmfixmshW_SeIs0xHj97gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0gdwqe6gATBfagbnBItpKKLXWV0U_MYk2qIuJRygurGwLtoEyxEa7HooetQk6_oPEw/Tensorflow+Banner+SS-01.png)